lib/elixir_jobs_web/controllers/offer_controller.ex

```diff
@@ -34,7 +34,8 @@ defmodule ElixirJobsWeb.OfferController do
 
   def create(conn, %{"offer" => offer_params}) do
     case Offers.create_offer(offer_params) do
-      {:ok, _offer} ->
+      {:ok, offer} ->
+        send_creation_email(offer)
```

Comment:
I'd just call `ElixirJobsWeb.Email.notification_offer_created_html(offer)` directly here, as in the `send_creation_email` there is no extra logic.